core/src/SYCL/Kokkos_SYCL_TeamPolicy.hpp

```diff
@@ -333,14 +333,15 @@ class Kokkos::Impl::TeamPolicyInternal<Kokkos::SYCL, Properties...>
   template <class FunctorType>
   int internal_team_size_recommended_for(const FunctorType& f) const {
     // FIXME_SYCL improve
```

Comment:
@masterleinad was the approve about the bit manipulation or about the value being returned?

Reply:
It was about the value returned.

Reply:
OK so we want to keep it.